tools/pipelines/build-client.yml

```diff
@@ -25,7 +25,7 @@ parameters:
 - name: buildToolsVersionToInstall
   displayName: Fluid build tools version (default = installs version in repo)
   type: string
-  default: latest
+  default: next
```

Comment:
Also curious why this. Not sure if `latest` mean latest from `main`, or latest from `lts`, but in either case I don't get a quick read on why `next` is the new correct value.

Reply:
It's an npm release tag. The code installs it using `npm i -g package@next`. It's no longer needed, since I released build-tools so latest has the right stuff. I'll revert this.